app/services/user.js

```diff
@@ -14,3 +15,17 @@ exports.signUp = async body => {
     throw error;
   }
 };
+
+exports.signIn = async ({ email, password }) => {
+  try {
+    const user = await User.getUser({ email });
+    if (user && (await compareHash(password, user.dataValues.password))) {
+      logger.info(`${user.dataValues.email} sign in correctly`);
+      return generaToken(user.dataValues);
+    }
+    logger.info(`wrong sign in user: ${email}`);
```

Comment:
typo(?)

Reply:
I don't understand your comment can be more specific